DATA_DICTIONARY.md

```diff
@@ -0,0 +1,71 @@
+# Data Dictionary for Super DCA Metrics
+
+### Average Price Impact
```

Comment:
You can make tables for this information vs. repeating "query title""chart name" "chart description" so many times.

Reply:
also, as a table it'll be clearer which charts show multiple metrics shown on them.